en/docs/install-and-setup/upgrading-the-solution/upgrading-wso2-api-manager-320.md

```diff
@@ -0,0 +1,61 @@
+# Upgrading to WSO2 API Manager 3.2.0
+
+WSO2 API Manager 3.1.0 is a base product of WSO2 Open Banking 2.0. This section instructs you on how to upgrade the API
+Manager to 3.2.0, which is a prerequisite for upgrading to API Manager 4.0.
+
+- Upgrade **IS as Key Manager 5.10.0** for API Manager 3.2.0 by following 
+[Upgrading WSO2 IS as Key Manager](https://apim.docs.wso2.com/en/3.2.0/install-and-setup/upgrading-wso2-is-as-key-manager/upgrading-from-is-km-5100-to-is-5100/).
+
+- Upgrade API Manager from 3.1.0 to 3.2.0 by following the 
+[API Manager documentation](https://apim.docs.wso2.com/en/3.2.0/install-and-setup/upgrading-wso2-api-manager/upgrading-from-310-to-320/#upgrading-api-manager-from-310-to-320).
+    
+    !!! note
+        In the above documentation, under [Step 1 - Migrate the API Manager configurations](https://apim.docs.wso2.com/en/3.2.0/install-and-setup/upgrading-wso2-api-manager/upgrading-from-310-to-320/#step-1-migrate-the-api-manager-configurations),
+        skip steps 6,7,8, and 9 as we are only trying to migrate the databases at this level.
+
+- Set up **IS 5.11.0 as Key Manager** for API Manager 3.2.0 
+
+    1. Download and install the WSO2 Identity Server 5.11.0 distribution from [here](https://wso2.com/identity-server/). 
+    2. Extract the downloaded archive file. This document refers to the root folder of the extracted file as `<IS_HOME>`.
+    3. Download the `wso2-obiam-accelerator-3.0.0.zip` file and extract it to the `<IS_HOME>` directory. 
+    4. Download the latest updates for `wso2-obiam-accelerator-3.0.0`. For more information, see [Getting WSO2 Updates](../setting-up-servers.md#getting-wso2-updates).
+    5. Open the `<IS_HOME>/<IS_ACCELERATOR_HOME>/repository/conf/configure.properties` file and update the hostnames 
+       and database details. These database configurations should point to the databases of Open Banking 2.0.
+    6. Go to the `<IS_HOME>/<IS_ACCELERATOR_HOME>/bin` directory and issue the following commands:
+           1. Run the merge.sh script.
+  
+             ```
+             ./merge.sh
+             ```
+  
+           2. Run the configure.sh script
```

Comment:
We don't recommend running the configure.sh command. @KalanaDananjaya why do we add this to the doc?

Reply:
Removing the steps to run configure.sh

Reply:
Fixed in 974508e9f4420db237b6b0beb4ca349c523a2c89